### PR TITLE
feat: Extract quality

### LINKED ISSUE #5: Support quality?
it seems like this library can't extract the torrent quality from its name (eg. CAM, MD, TS, HDTS etc).

It would be cool to have this feature.

### PR DESCRIPTION
**Pull request by FezVrasta**
Fixes #5

### CODE REVIEW

File: src/handlers.js [hunk 1]
```diff
@@ -85,4 +85,21 @@ exports.addDefaults = /** @type Parser */ parser => {
...
+    parser.addHandler("quality", /\bDVDRip\b/i, { type: "lowercase" });
+    parser.addHandler("quality", /\bDVDscr\b/i, { type: "lowercase" });
+    parser.addHandler("quality", /\b(?:HD\-?)?TVRip\b/i, { type: "lowercase" });
+    parser.addHandler("quality", /\bTC\b/i, { type: "lowercase" });
+    parser.addHandler("quality", /\bPPVRip\b/i, { type: "lowercase" });
+    parser.addHandler("quality", /\bR5\b/i, { type: "lowercase" });
+    parser.addHandler("quality", /\bVHSSCR\b/i, { type: "lowercase" });
+    parser.addHandler("quality", /\bBluray\b/i, { type: "lowercase" });
+    parser.addHandler("quality", /\bWEB\-?DL\b/i, { type: "lowercase" });
+    parser.addHandler("quality", /\bWEB\-?Rip\b/i, { type: "lowercase" });
```
Comment (FezVrasta) [inline src/handlers.js]: There are probably more but I think this is a good start already

**Comment by clement-escolano** [comment]
Hello!
About this one, what do you think about the `source` field returned by the module (the regex is [here](https://github.com/clement-escolano/parse-torrent-title/blob/master/src/handlers.js#L41)) ?
If you think the term `quality` is more appropriate than `source` then it is possible to create a handler that will copy the `source` into `quality`. But I guess it is more appropriate to improve the existing `source` handler than creating an alternate `quality` handler.
Thanks!

PS: for information, the handler that will copy the `source` in `quality` should look like this:
```js
paser.addHandler(({result}) => { result.quality = result.source });
```

**Comment by FezVrasta** [comment]
I didn't noticed there was already an handler for this. I'll change my PR to extend it then

**Comment by clement-escolano** [comment]
I think you don't need to escape the `-` character in the regex (as ESLint [tells us](https://travis-ci.org/clement-escolano/parse-torrent-title/jobs/324572419)).
Also, can you add these regex (some tests [are failing](https://travis-ci.org/clement-escolano/parse-torrent-title/jobs/324572416)):

```js
parser.addHandler("source", /\bHDTV\b/i, { type: "lowercase" });
parser.addHandler("source", /\b(DVD|DVDR[0-9])\b/i, { type: "lowercase" });
```

File: src/handlers.js [hunk 2]
```diff
@@ -38,7 +38,25 @@ exports.addDefaults = /** @type Parser */ parser => {
     parser.addHandler("container", /\b(MKV|AVI|MP4)\b/i, { type: "lowercase" });
 
     // Source
-    parser.addHandler("source", /hdtv|bluray|(?:b[dr]|dvd|hd|tv)rip|web-?(?:dl|rip)|dvd|ppv/i, { type: "lowercase" });
+    parser.addHandler("source", /\b(?:HD\-?)?CAM\b/i, { type: "lowercase" });
+    parser.addHandler("source", /\b(?:HD\-?)?T(?:ELE)?S(?:YNC)?\b/i, {
+        type: "lowercase",
```
Comment (clement-escolano) [inline src/handlers.js]: Can you inline the options please? (I know ESLint should have caught this)

**Comment by clement-escolano** [comment]
Merged!
Sorry for the delay and thank you!

**Pull request merged**